rdrf/rdrf/template_loaders/translation.py

```diff
@@ -0,0 +1,46 @@
+import logging
+from django.template import Template
+from django.template import Origin
+from django.template import TemplateDoesNotExist
+from django.template.loaders.base import Loader as BaseLoader
+from rdrf.utils import get_registry_definition_value
+
+logger = logging.getLogger(__name__)
+
+
+class Loader(BaseLoader):
+    """
+    A custom template loader which retrieves embedded html templates from the registry definition
+    models.
+    """
+    PREFIX = "rdrf://"
+    PRELUDE = "{% load i18n %}"
+    
+    def get_contents(self, origin):
+        name = origin.name
+        if not name.startswith(self.PREFIX):
```

Comment:
Just a thought - if the template'path' is an URI, then why not parse as a URI (to validate) and check the scheme is 'rdrf'. self.PREFIX becomes self.SCHEME?